File: Week 1/fizz_buzz.py
class Solution(object):
    def fizzBuzz(self, n):
        """
        :type n: int
        :rtype: List[str]
        """
        str = []
        
        for i in range(1 , n+1 , 1):
            if(i % 3 == 0):
                if(i % 5 == 0):
                    str.append("FizzBuzz")
                else:
                    str.append("Fizz")
            elif(i % 5 == 0):
                str.append("Buzz")
            else:
                str.append("{}".format(i))
        return str
                    
        

File: Week 1/fizz_buzz.cpp
class Solution {
public:
    vector<string> fizzBuzz(int n) {
        vector<string> str;
        for(int i = 1; i <= n ; i++ ){
            if(i % 3 == 0){
                if(i % 5 == 0){
                    str.push_back("FizzBuzz") ;
                }else{
                    str.push_back("Fizz");
                }
            }else if(i % 5 == 0) {
                str.push_back("Buzz");
            }else{
                str.push_back(to_string(i));
                
            }
        }
        return str;
    }
};